tensorflow/lite/micro/tools/make/targets/mcu_riscv_makefile.inc

```diff
@@ -1,69 +1,70 @@
 # Settings for RISCV 32-bit MCU toolchain.
-ifeq ($(TARGET), riscv32_mcu)
-  TARGET_ARCH := riscv32_mcu
-  TARGET_TOOLCHAIN_PREFIX := riscv64-unknown-elf-
+TARGET_ARCH := riscv32_mcu
+TARGET_TOOLCHAIN_PREFIX := riscv64-unknown-elf-
 
-  $(eval $(call add_third_party_download,$(RISCV_TOOLCHAIN_URL),$(RISCV_TOOLCHAIN_MD5),riscv_toolchain,))
-  $(eval $(call add_third_party_download,$(SIFIVE_FE310_LIB_URL),$(SIFIVE_FE310_LIB_MD5),sifive_fe310_lib,))
+export PATH := $(MAKEFILE_DIR)/downloads/riscv_toolchain/bin/:$(PATH)
 
-  PLATFORM_FLAGS = \
-    -march=rv32imac \
-    -mabi=ilp32 \
-    -mcmodel=medany \
-    -mexplicit-relocs \
-    -fno-builtin-printf \
-    -fno-exceptions \
-    -DTF_LITE_MCU_DEBUG_LOG \
-    -DTF_LITE_USE_GLOBAL_CMATH_FUNCTIONS \
-    -fno-unwind-tables \
-    -ffunction-sections \
-    -fdata-sections \
-    -funsigned-char \
-    -Wvla \
-    -Wall \
-    -Wextra \
-    -Wsign-compare \
-    -Wdouble-promotion \
-    -Wshadow \
-    -Wunused-variable \
-    -Wmissing-field-initializers \
-    -Wno-unused-parameter \
-    -Wno-write-strings \
-    -Wunused-function \
-    -fno-delete-null-pointer-checks \
-    -fno-threadsafe-statics \
-    -fomit-frame-pointer \
-    -fno-use-cxa-atexit \
-    -Os
+$(eval $(call add_third_party_download,$(RISCV_TOOLCHAIN_URL),$(RISCV_TOOLCHAIN_MD5),riscv_toolchain,))
+$(eval $(call add_third_party_download,$(SIFIVE_FE310_LIB_URL),$(SIFIVE_FE310_LIB_MD5),sifive_fe310_lib,))
 
-  CXXFLAGS += $(PLATFORM_FLAGS) \
-    -fpermissive \
-    -fno-rtti \
-    --std=gnu++11
+PLATFORM_FLAGS = \
```

Comment:
Many of the flags are now consolidated into the top-level makefile:
https://github.com/tensorflow/tflite-micro/blob/bfd9f923c7d6c1a998c6077cf46bdece46ddaf65/tensorflow/lite/micro/tools/make/Makefile#L130-L153

It might be worth shortening the riscv list to only be target-specific, for example here is the bluepill PLATFORM_FLAGS:
https://github.com/tensorflow/tflite-micro/blob/bfd9f923c7d6c1a998c6077cf46bdece46ddaf65/tensorflow/lite/micro/tools/make/targets/bluepill_makefile.inc#L25-L32

Reply:
Good idea, I've shrunk the number of flags down.